### PR TITLE
Add document for simulcast

### PR DESCRIPTION
**Pull request by starwarfan**
c46d20eba3019f0c7ea9e2a6f1269a6844a6d80e

### CODE REVIEW

File: docs/mdfiles/simulcast.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+OWT Simulcast Description
+---------------------
+
+# {#simulcast}
+The MCU server supports simulcast. This can be enabled through OWT Client SDK API. <br>
```
Comment (jianjunz) [inline docs/mdfiles/simulcast.md]: MCU -> conference. It also supports forwarding mode.

`<br>` is not required.

File: docs/mdfiles/simulcast.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+1. Publish a simulcast stream
+~~~~~~{.js}
+//Example of simulcast publication
```
Comment (jianjunz) [inline docs/mdfiles/simulcast.md]: Space after `//`. Ends with period.

File: docs/mdfiles/simulcast.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+conference = new Owt.Conference.ConferenceClient();
+// ...
+conference.join(token).then(resp => {
+    // ...
+    Owt.Base.MediaStreamFactory.createMediaStream(new Owt.Base.StreamConstraints(
+            audioConstraints, videoConstraints)).then(stream => {
+        /*
+         * Use `VideoEncodingParameters` as publish option.
```
Comment (jianjunz) [inline docs/mdfiles/simulcast.md]: VideoEncodingParameters -> RTCRtpEncodingParameters. Please also provide a link to its definition, i.e.: https://w3c.github.io/webrtc-pc/#dom-rtcrtpencodingparameters.

File: docs/mdfiles/simulcast.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            stream, new Owt.Base.StreamSourceInfo(
+                'mic', 'camera'));
+        conference.publish(localStream, publishOption, codecs).then(publication => {
+            // ...
+        });
+    });
+~~~~~~
+
+2. Subscribe a simulcast stream
+//Example of subscription
```
Comment (jianjunz) [inline docs/mdfiles/simulcast.md]: Ditto.

File: docs/mdfiles/simulcast.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    conference.subscribe(remoteStream, subscribeOption).then((subscription) => {
+        // ...
+    });
+~~~~~~
+
+> **Note:**
+1. The simulcast stream published to conference won't be transcoded.
+2. The `rid` attribute may not be present once a 'streamadded' event triggered. Users should listen on stream's `updated`
+event for new `rid` added.
+3. Current browsers(Chrome/Firefox/Safari) support VP8 simulcast. Only software encoder in chrome supports H264 simulcast.
```
Comment (jianjunz) [inline docs/mdfiles/simulcast.md]: Does Safari supports VP8 simulcast? AFAIK, Safari supports H.264 simulcast.

**Comment by jianjunz** [comment]
Fix #295.

File: docs/mdfiles/simulcast.md (hunk 4 above)
Comment (yanbin7) [reply docs/mdfiles/simulcast.md]: {rid: 'q', active: true}  update to {rid: 'q', active: true, scaleResolutionDownBy: 1.0}

File: docs/mdfiles/simulcast.md (hunk 5 above)
Comment (yanbin7) [reply docs/mdfiles/simulcast.md]: Safari support VP8 simulcast.
And Do we need add description about there is one rid for remote stream with Firefox H264 simulcast

File: docs/mdfiles/simulcast.md (hunk 1 above)
Comment (lzhai) [reply docs/mdfiles/simulcast.md]: This can be enabled through OWT Client SDK for JavaScript. Only JavaScript SDK can enable simulcast.

**Pull request merged**